libraries/cloudharness-common/tests/test_infrastructure.py

```diff
@@ -2,6 +2,8 @@
 set_test_environment()
 
 from cloudharness.infrastructure import k8s
-
+kubectl_enabled = False
```

Comment:
This violates PEP8, there must be a newline between variable module level function

Reply:
Did this and other formatting fixes